bin/templates/scripts/cordova/lib/check_reqs.js

```diff
@@ -49,24 +49,18 @@ const COCOAPODS_REPO_NOT_FOUND_MESSAGE = 'The CocoaPods repo at ~/.cocoapods was
  * Checks if xcode util is available
  * @return {Promise} Returns a promise either resolved with xcode version or rejected
  */
-module.exports.run = module.exports.check_xcodebuild = function () {
-    return checkTool('xcodebuild', XCODEBUILD_MIN_VERSION, XCODEBUILD_NOT_FOUND_MESSAGE);
-};
+module.exports.run = module.exports.check_xcodebuild = () => checkTool('xcodebuild', XCODEBUILD_MIN_VERSION, XCODEBUILD_NOT_FOUND_MESSAGE);
```

Comment:
I think I have an unpopular opinion here, but as much as I love arrow functions for it's automatic context binding, I really hate the syntax and find it confusing. In my own code bases, I always enforce parenthesis and curly braces for consistency. In my opinion, this makes the code base easier to read.

```javascript
() => {
    ...
}
```

Omitting the curly braces for the implicit return is, in my opinion confusing. Additionally it does not work if you're attempting to return an object literal. Therefore you are forced to switch up your syntax style depending on what the type of data you are returning and could be a source of bugs. Considering it doesn't even produce a syntax error, it can be a bug that may be hard to trace as it will simply just cause the function to return undefined. See below.

```
Welcome to Node.js v12.13.1.
Type ".help" for more information.
> let x = () => {test:true}
undefined
> x()
undefined
> x
[Function: x]
> let y = () => 123
undefined
> y()
123
> 
```

Not sure on the capabilities of `lebab`, but I would prefer configuring it to always use parenthesis for defining the arrow function argument section and to always use curly brace to define the function body, if possible.

Reply:
I wouldn't be a fan of always requiring the parens, but I agree that some expressions involving arrow functions can become hard to read. The code I flagged here was a good example for that.

Reply:
The function body with curly braces is really my primary concern.